src/OSPSuite.Core/Services/ExplicitFormulaParser.cs

```diff
@@ -71,6 +71,23 @@ public void Parse()
             throw new FuncParserException($"Unable to parse expression: {ex.Message}", ex);
          }
       }
+
+      public (double value, bool success) TryCompute(double[] variableValues, double[] parameterValues)
+      {
+         var success = _parserFunc.TryParse();
```

Comment:
maybe we do need a TryCompute in FuncParser...This is good for now I think but this seems to be a bit backward

Reply:
Can do. TryCompute would behave exactly as your current C# implementation: call TryParse and return false if TrParse fails; otherwise return true and the result of the calculation (which could also be Nan, Infinity etc.)